src/library.js

```diff
@@ -5162,6 +5162,9 @@ LibraryManager.library = {
   },
   // =======================================================================
 
+  __handle_stack_overflow: function() {
```

Comment:
`handle` -> `abort`?

Reply:
I don't know if `__abort_stack_overflow` is really a good name for the handler function. Binaryen gives the runtime flexibility to do whatever in face of stack overflow. Hypothetically, a runtime could implement a mechanism similar to `sigaltstack(2)` to handle stack overflows.

Reply:
Hmm, now that I look closer at this - does the wasm backend directly generate a call to this function? If so, then the name is very generic. How about `__wasm_stack_overflow`? Having compiler backend construct a function call with such a name that does not connect to an established API would make the symbol look somewhat floating.

Reply:
The WASM backend does not generate a call to this. `wasm-emscripten-finalize` in binaryen inserts a call to this function when passed a certain flag. So the `__wasm` prefix is inappropriate.